ansible/roles/host-ocp4-provisioner/tasks/ec2_prereqs.yml

```diff
@@ -0,0 +1,63 @@
+- name: Get awscli bundle
```

Comment:
Something we can consider installing via pip into venv and pin to a version? Because it never breaks when we use whatever the latest is...

Reply:
I was thinking the same.